### PR TITLE
Completed es6-classes- Devan Rivera 

### PR DESCRIPTION
**Pull request by devanrivera98**
Screen Shot 2023-03-24 at 4 39 19 PM
Screen Shot 2023-03-24 at 4 39 26 PM

### CODE REVIEW

File: es6-classes/circle.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+class Circle extends Shape {
+  constructor(area, circumference, radius) {
+    super(area, circumference);
+    this.radius = radius;
+  }
+
+  print() {
+    console.log(`The radius of the Circle is ${this.radius}`);
```
Comment (RobertGardner) [inline es6-classes/circle.js:10]: You need to call super.print() to get the Shape's properties.
This should return a string, not log directly.

File: es6-classes/shape.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* exported Shape */
+class Shape {
+  constructor(area, circumference) {
+    this.area = area;
+    this.circumference = circumference;
+  }
+
+  print() {
```
Comment (RobertGardner) [inline es6-classes/shape.js:8]: Return a string.

File: es6-classes/square.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Square */
+class Square extends Shape {
+  constructor(area, circumference, width) {
+    super(area, circumference);
+    this.width = width;
+  }
+
+  print() {
```
Comment (RobertGardner) [inline es6-classes/square.js:9]: Same as for Circle.

**Review by RobertGardner — CHANGES_REQUESTED** [review]

**Comment by devanrivera98** [comment]
Screen Shot 2023-03-25 at 4 17 05 PM
Screen Shot 2023-03-25 at 4 17 10 PM

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**